tests/test_deprecation.py

```diff
@@ -55,6 +56,7 @@ def test_deprecation_warnings_are_accompanied_by_appropriate_comment(
         ), f"Either a 'deprecated-v...' comment or 'DeprecationWarning' call is missing from {file}"
 
 
+@pytest.mark.unit
```

Comment:
This module also looks like it could be marked as unit.
https://github.com/great-expectations/great_expectations/pull/8275/files#r1259817234

Reply:
👍🏼